map_analysis_2d/workers/peak_fitting_worker.py

```
@@ -1,161 +1,183 @@
+                        rmse = np.sqrt(ss_res / n_pts) if n_pts > 0 else 0.0
+                        if amp_indices:
+                            max_amp = max(float(popt[i]) for i in amp_indices)
+                            results['snr'][pos_key] = max_amp / rmse if rmse > 0 else (np.inf if max_amp > 0 else 0.0)
+                        else:
+                            results['snr'][pos_key] = 0.0
+
+                        for param_index, name in enumerate(param_names):
+                            results['map_parameters'][name][pos_key] = popt[param_index]
+                        results['r_squared'][pos_key] = r_squared
+
+                        popt_dict = dict(zip(param_names, popt))
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)